sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -36,6 +36,21 @@ SPV_MATRIX_LAYOUT_TRAITS(matrix_layout::col_major,
 SPV_MATRIX_LAYOUT_TRAITS(matrix_layout::packed_a, __spv::MatrixLayout::PackedA)
 SPV_MATRIX_LAYOUT_TRAITS(matrix_layout::packed_b, __spv::MatrixLayout::PackedB)
 
+enum class matrix_use { matrix_a, matrix_b, accumulator, unnecessary };
+
+template <matrix_use Use> struct spv_matrix_use_traits {
+  static constexpr __spv::MatrixUse value = __spv::MatrixUse::MatrixA;
```

Comment:
same as above, this should be empty.

Reply:
you mean we should remove the default value. but we have default value for matrix_layout's traits

Reply:
you mean we should remove the default value. but we have default value for matrix_layout's traits